front/app/api/analysis_summary_pre_check/useAddAnalysisSummaryPreCheck.ts

```diff
@@ -0,0 +1,22 @@
+import { useMutation } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import { ISummaryPreCheck, ISummaryPreCheckAdd } from './types';
+
+const addAnalysisSummaryPreCheck = async ({
+  analysisId,
+  filters,
+}: ISummaryPreCheckAdd) =>
+  fetcher<ISummaryPreCheck>({
+    path: `/analyses/${analysisId}/summaries/pre_check`,
+    action: 'post',
```

Comment:
I started with making this a GET request, and while implementing I changed it to `POST` because
- From a RESTful point of view, post make more sense (you're 'creating' a new check)
- Another more pressing reason, which I forgot.

It's working, so will just keep it that way to be able to move on.